File: mssql/jruby/read/mssql_read.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	read/mssql_read.rb
#					Jan/17/2013
#
# --------------------------------------------------------
include Java
import com.microsoft.sqlserver.jdbc.SQLServerDriver
import java.sql.DriverManager
import java.lang.System
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
# --------------------------------------------------------
puts "*** 開始 ***"
str_connect = "jdbc:sqlserver://host_mssql;databaseName=city;"

conn = DriverManager.getConnection(str_connect,"sa", "scott_tiger")

sql_read_proc(conn)

conn.close()

puts "*** 終了 ***"
# --------------------------------------------------------

File: mssql/jython/read/mssql_read.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	mssql_read.py
#					Jan/17/2013
#
# ----------------------------------------------------------------
import sys
from java.lang import	System
#
import java
from java.sql import DriverManager
#
# ----------------------------------------------------------------
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_rdb_manipulate import display_proc
# ----------------------------------------------------------------
System.out.println ("*** start ***")
print ("*** 開始 ***")
#
url="jdbc:sqlserver://host_mssql;databaseName=city;"
user = "sa"
password = "scott_tiger"

conn = DriverManager.getConnection (url,user, password)

display_proc (conn)
#
conn.close()
print ("*** 終了 ***")
#
# ----------------------------------------------------------------
